README.md

```diff
@@ -153,6 +157,9 @@ class PhlexIcons < Phlex::HTML
       Radix::Home(class: 'size-4')
       Remix::HomeLine(class: 'size-4')
       Tabler::Home(variant: :filled, class: 'size-4')
+
+      # or with a string
+      Icon("bootstrap/house", class: '...')
```

Comment:
For consistency, could you please use `class: 'size-4'` or convert the other examples to use `class: '...'`.